explorer/templates/explorer/query_list.html

```diff
@@ -6,15 +6,15 @@
     {% if recent_queries|length > 0 %}
         <div class="container">
             <h3>
-                {% blocktrans trimmed with qlen=recent_queries|length %}
+                {% blocktranslate trimmed with qlen=recent_queries|length %}
```

Comment:
Might be worth looking over the history of this heading and finding out what happened to `qlen`. Maybe it can be removed and make this just a `{% translate` tag

Reply:
Ah yes - it can definitely be removed. I simplified the heading text and forgot to remove this.

Reply:
Shall I revert it to `translate` and is there a different course of action to solve `qlen`? @marksweb @chrisclark

Reply:
@shiyanshirani Yeah just make it `{% translate "Recently Run by You" %}`

Reply:
bingo. @shiyanshirani if you can make that last change, I will do a final test, and we can get this merged!

Reply:
I've pushed the changes.